crates/bevy_render/src/shader/shader.rs

```diff
@@ -27,24 +27,47 @@ pub enum ShaderError {
     #[error("Shader compilation error:\n{0}")]
     Compilation(String),
 
-    #[cfg(any(target_os = "ios", all(target_arch = "aarch64", target_os = "macos")))]
+    #[cfg(not(any(
+        target_arch = "wasm32",
+        all(target_arch = "x86_64", target_os = "linux", target_env = "gnu"),
+        all(target_arch = "x86_64", target_os = "macos"),
+        all(target_arch = "aarch64", target_os = "android"),
+        all(target_arch = "armv7", target_os = "android"),
```

Comment:
```suggestion
        all(target_arch = "armv7", target_os = "androidabi"),
```
This must be used, to fix the CI